docs/dev_guides/settings_validation.md

```diff
@@ -0,0 +1,142 @@
+## Settings Schema Validation
+
+[Schema validation](https://github.com/moj-analytical-services/splink/blob/master/splink/validate_jsonschema.py) is currently performed inside the [settings.py](https://github.com/moj-analytical-services/splink/blob/master/splink/settings.py#L44C17-L44C17) script.
+
+This assesses the user's settings dictionary against our custom [settings schema](https://github.com/moj-analytical-services/splink/blob/master/splink/files/settings_jsonschema.json). Where the data type, key or entered values devivate from what is specified in the schema, an error will be thrown.
```

Comment:
Swap these two lines around - I think it is important to explain what a thing is before specifying where it is performed.